app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -147,7 +147,7 @@ def tweeter
     end
 
     def presenter_types
-      Hyrax.config.registered_curation_concern_types.map(&:underscore) + ["collection"]
+      Hyrax.config.registered_curation_concern_types.map(&:underscore) + ["collection", "Collection"]
     end
 
     # @return [Array] presenters grouped by model name, used to show the parents of this object
```

Comment:
What about instead of adding `Collection` to `#presenter_types` you change `#grouped_presenters` to downcase the exceptions like it does for filtering?
```
grouped.reject! { |obj| except.include? obj.downcase } unless except.nil?
```
Or would that potentially break things unless the downcasing is applied to both sides?
```
grouped.select! { |obj| obj.downcase == filtered_by.downcase } unless filtered_by.nil?
grouped.reject! { |obj| except.map(&:downcase).include? obj.downcase } unless except.nil?
```

Reply:
Looking at that now.